src/main/resources/interface-specification.yml

```diff
@@ -245,6 +245,56 @@ paths:
             application/problem+json:
               schema:
                 $ref: '#/components/schemas/Problem'
+  '/agreements/{agreement_id}/document':
```

Comment:
we generally use camel case for variables

Reply:
Yes, this is a missing commit.
I found the same problem in route `/agreements/{agreement_id}`.
I'm fixing them.